README.md

```diff
@@ -264,15 +273,6 @@ THEN:
 	I store the value of body path (.*) as (.*) in global scope
 ```
 
-The simplest way to adopt these expressions is to create a file named apickli-gherkin.js in features/step_definitions and extend the apickli/gherkin.js module.
-
-add the following to test/features/step_definitions/apickli-gherkin.js
-```javascript
-module.exports = require('apickli/apickli-gherkin');
-```
-
-If using Windows, follow this guide to create a symlink: [How-To Geek Guide](http://www.howtogeek.com/howto/16226/complete-guide-to-symbolic-links-symlinks-on-windows-or-linux/).
```

Comment:
Do we still need symlinking now that we have a way of actually requiring the apickli-gherkin module?